src/test/java/jenkins/plugins/git/GitToolChooserTest.java

```diff
@@ -637,6 +655,18 @@ private void buildAProject(GitSampleRepoRule sampleRepo, boolean noCredentials)
         }
     }
 
+    /* Attempt to perform a checkout without defining a remote repository. Expected to fail, but should not report NPE */
+    private void failAProject(GitSampleRepoRule sampleRepo) throws Exception {
+        WorkflowJob p = jenkins.jenkins.createProject(WorkflowJob.class, "intentionally-failing-job-without-remote-config");
+        p.setDefinition(new CpsFlowDefinition("node {\n"
+                                              + "  checkout(\n"
+                                              + "    [$class: 'GitSCM'] \n"
```

Comment:
Remove useless trailing space
```suggestion
                                              + "    [$class: 'GitSCM']\n"
```